homework23_SQl/src/main/resources/init.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE Homework (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(50) NOT NULL,
+description TEXT,
+PRIMARY KEY (id)
+);
+
+CREATE TABLE Lesson (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(50) NOT NULL,
+updatedAt DATE,
+homework_id INT NOT NULL,
+PRIMARY KEY (id),
+FOREIGN KEY (homework_id) REFERENCES Homework(id)
+);
+
+CREATE TABLE Schedule (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(50) NOT NULL,
+updatedAt DATE,
+PRIMARY KEY (id)
+);
+
+CREATE TABLE ScheduleLesson (
+id INT NOT NULL AUTO_INCREMENT,
```

Comment:
here is better to remove id column but add complex primary key from 2 values

something like I did on lesson 
```
create table hotel_visitors (
	hotelId int NOT NULL,
	visitorId int NOT NULL,
    primary key (hotelId, visitorId)
)
```